transport-classes-io_uring/src/main/java/io/netty/channel/uring/Native.java

```diff
@@ -340,6 +340,15 @@ static boolean isIOUringSupportSplice(int ringFd) {
         return ioUringProbe(ringFd, new int[] { Native.IORING_OP_SPLICE });
     }
 
+    /**
+     * check current kernel version whether support io_uring_register_io_wq_worker
+     * Available since 5.15.
+     * @return true if support io_uring_register_io_wq_worker
+     */
+    static boolean isRegisterIOWQWorkerSupported() {
+        return Native.checkKernelVersion(Native.kernelVersion(), 5, 15);
```

Comment:
I think there should be a better way how to check this... let me do some research

Reply:
Looking forward to your good news